app/controllers/content_types/school_home_pages_controller.rb

```diff
@@ -33,7 +33,7 @@ def slideshow
         'COMMON STYLES' => cascade_block('_cascade/blocks/html/common_styles'),
         'COMMON SCRIPTS' => cascade_block('_cascade/blocks/html/common_scripts'),
         'CUSTOM THEME' => cascade_block('_cascade/blocks/custom_themes/law'),
-        'GLOBAL FOOTER' => cascade_block('_cascade/blocks/html/law_footer'),
+        'GLOBAL FOOTER' => cascade_block('widgets/shared/law_footer'),
```

Comment:
Is the block being moved in Cascade? The idea with these controllers is that the partials hierarchy mirrors what's in Cascade.